ts/components/session/settings/SessionSettingListItem.tsx

```diff
@@ -88,7 +88,7 @@ export class SessionSettingListItem extends React.Component<Props, State> {
             />
           )}
 
-          {type === SessionSettingType.Slider && (
+          {type === SessionSettingType.Slider && title === 'Message TTL' ? (
```

Comment:
These titles should be put in an enum instead.

Reply:
Actually a much better thing would be to utilize the `content` object to set specific options like it's done above in `Button` and `Option`